translations/messages.ru.yaml

```diff
@@ -376,8 +376,7 @@ kbin_intro_title: Посмотреть Федиверс
 kbin_intro_desc: является децентрализованной платформой для агрегирования контента
   и микроблогинга, которая работает в сети Fediverse.
 kbin_promo_title: Создайте ваш инстанс
-kbin_promo_desc: '%Начальная ссылка%Клонировать репозиторий%Конечная ссылка% и развивайте
-  Федиверс'
+kbin_promo_desc: 'перейдите по ссылке и начните свой проект'
```

Comment:
It looks like this lost the link, I think there's some confusion on the tokens. 

If you take https://github.com/MbinOrg/mbin/blob/main/translations/messages.nl.yaml#L379 for instance, it has `%link_start%` and `%link_end%` around where the link is. These aren't meant to be translated, they are replacement tokens used to insert the proper HTML

Reply:
oops my bad on the approval, @gregscoor can you address this?

Reply:
madved says if we insert %begin_link% in this string weblate warnings about uncompleted translate

Reply:
as for me, so I would remove this line from the translation